nncf/experimental/onnx/graph/onnx_graph.py

```diff
@@ -0,0 +1,77 @@
+import onnx
+
+
+# pylint: disable=no-member
+
+class ONNXGraph:
```

Comment:
Make sure that every public method has a docstring and that every method has both argument and return type hints.

Reply:
Done